### DISCUSSION TITLE
Can't Access GitHub Pages

### DISCUSSION THREAD

**Discussion by kcimartinez** [community/community#27757] [Other Feature Feedback, Questions, & Ideas] [answered]
I can't seem to access any GitHub pages since yesterday even the ones I have previously visited. More generally, I can't access github.io. I'm from the Philippines if that helps. I tried switching browsers and even connecting to github.io using mobile data to no avail. Although I can access github.com.

![githubpages](https://user-images.githubusercontent.com/107381315/182006688-5016ba4c-a2bd-4101-a174-eb764e81dc48.png)

**Comment by jfullstackdev** [comment]
there is no problem on GitHub Pages 

![image](https://user-images.githubusercontent.com/47092464/182010916-46244e24-cee0-472d-a915-714e4c7cf234.png)

can you send the link, so I can check it too ?

**Reply by kcimartinez** [reply]
I can't access any site with github.io in its url. You can look at sample websites here in https://www.theodinproject.com/lessons/node-path-javascript-weather-app

Scroll to the bottom part and click on "Live preview" on any of the projects. I can't access any of them.

I tried using Brave browser's free VPN and I can access the sites pretty well. I guess this has something to do with my region.

**Reply by jfullstackdev** [reply]
why are you using another external service ? can you send a direct link of a repo with GitHub Pages enabled, so I can see what's happening

**Reply by kcimartinez** [reply]
Okay, here is one of my projects that I was able to access before yesterday https://muneyy.github.io/TOP-ToDolist/

**Reply by jfullstackdev** [reply]
that's working fine 

![image](https://user-images.githubusercontent.com/47092464/182012810-396f394f-bc96-4ce3-803f-a594615904bc.png)

**Reply by kcimartinez** [reply]
Yeah I think the problem is my region. With VPN I can access github.io sites.

**Reply by jfullstackdev** [reply]
I'm in Philippines too, not using VPN

**Reply by kcimartinez** [reply]
Thanks for your help, it's working now.

**Comment by MrSarno** [comment]
Hey @Muneyy,

Try flushing your DNS cache if you haven't already.

There's a step-by-step guide for the most commonly used Operating Systems [on this page](https://www.hostinger.com/tutorials/how-to-flush-dns#How_to_Flush_DNS_Cache).

**Reply by kcimartinez** [reply]
Thanks for the help, it's working now.

**Comment by innovatus-pub** [comment]
I am also experiencing the same issues with github.io pages.  https://innovatus-pub.github.io is the website but I suspect it's an ISP or DNS issue.  I have tried flushing DNS cache but it doesn't seem to fix the problem.

**Comment by kcimartinez** [comment, marked as answer]
Hey everyone, as of the time of writing, I can access github.io again. I guess someone fixed it.

**Comment by glncy** [comment]
I'm having an issue also accessing GitHub pages. I am also in the Philippines. I don't know why. Tried flushing my DNS and such but no luck. I guess ISPs? :((

**Reply by munkeywolf00** [reply]
I am also having this issue. It might be an ISP issue.

**Reply by LiteBrite82** [reply]
@glncy 
I checked with the pages team to look for regional outages; all tests show that pages should be working in the Philippines. Check with your ISP and local connections.

If the problem persists, please create a new discussion in `Pages` so we can collect more community data on the issue.

**Reply by LiteBrite82** [reply]
Thanks for posting and supporting the community!!!!

**Comment by lealismo** [comment]
The pages are normal here

**Reply by git900000** [reply]
Similar incident here. I tried to open a community page. It gave the following error:

"Access to Github.com was denied. You don't have authorization to view this page. HTTP ERROR 403" 

![Screenshot 2024-10-22 11 55 19 AM](https://github.com/user-attachments/assets/5abe8905-c3de-4579-87e9-3d7e91ea94bd)

**Reply by git900000** [reply]
Also, I meant to reply to @MrSarno.

**Reply by git900000** [reply]
I apologize; I now can access it, but I would still appreciate an answer, as it might happen again in the future.

**Reply by MrSarno** [reply]
> I apologize; I now can access it, but I would still appreciate an answer, as it might happen again in the future.

Are / were you using a proxy or VPN?

**Reply by git900000** [reply]
No sir.
